### ISSUE TITLE
is it possible to make predictions on individual frames, instead of entire vedio

### ISSUE THREAD

**Issue by andro-galexy** [chen0040/keras-video-classifier-web-api#3]
From the sample predict script in the demo dir, it looks like VGG16 features of all frames in the video's  are extracted ones , and then it's feeded to pre-trained model to make a prediction, but i want to predict it for every individual frame in realtime, instead of proccessing entire video at once, will it affect the overall accuracy , or the  model only performs for the entire video as LSTM needs sequntial frames to process.

**Comment by yzheng97** [comment]
@swapgit You could use a sliding window. Say, extract the nearby 15 frame-wise features and feed it into the network.

Only my point of view.
